public/app/plugins/panel/annolist/AnnoListPanel.tsx

```diff
@@ -25,8 +36,10 @@ interface State {
   queryUser?: UserInfo;
   queryTags: string[];
 }
-
 export class AnnoListPanel extends PureComponent<Props, State> {
+  style = getStyles(config.theme);
+  subs: Unsubscribable[] = [];
```

Comment:
Use Subscription instead has an internal array, then no need to loop through this and call unsub as this is all handled by Subscription